src/main/resources/hudson/plugins/powershell/PowerShell/config.jelly

```diff
@@ -26,4 +26,9 @@ THE SOFTWARE.
   <f:entry title="${%Command}" description="${%description(rootURL)}">
     <f:textarea name="command" value="${instance.command}" class="fixed-width" />
   </f:entry>
+  
+  <f:entry field="stopOnError" title="Stop On Errors" description="Stops script when some step fails. Equivalent to Shell set -e. Translates to $ErrorActionPreference Stop or Continue">
```

Comment:
`ErrorActionPreference = "stop"` is not quite equivalent to `set -e`, as it only stops if a cmdlet fails, but it will still continue executing even when an executable exits with a non-zero exit code. There's an open issue for this one though. https://github.com/PowerShell/PowerShell/issues/3415

Reply:
reworded from Equivalent to Similar